### ISSUE TITLE
my zettlr couldn't export PDF and Word ,but I have install Pandoc ?why ?

### ISSUE THREAD

**Issue by zhangzhaoyue919** [Zettlr/Zettlr#1695]
An error occurred on export: pandoc "/var/folders/yk/ny......

**pandoc: $: openBinaryFIle:does not exist (No such file or directory)**

hope help !!

**Comment by jwagon** [comment]
I'm seeing the same bug. Working with Mac version 1.8.7, installed through downloaded disk image. pandoc 2.11.4 installed through Homebrew. Export to HTML produces this error:

![image](https://user-images.githubusercontent.com/908153/106570695-3c3f7b80-6504-11eb-826e-892f359d0c0d.png)

Any suggestions?

**Comment by VepryB** [comment]
I have a similar problem! I had a problem when I want to export the markdown into anything. I've tried exporting to pdf, word, and etc.. Working with Windows version 1.8.7. I downloaded pandoc using pandoc-2.11.4-windows-x86_64.msi

Export to Word produces this error:
![Screenshot 2021-03-02 132214](https://user-images.githubusercontent.com/72779583/109586952-5c417980-7b5a-11eb-8f2e-52bb6e9a12b2.png)

It said "pandoc: $citeproc$: openBinaryFile: does not exist (No such file or directory)"
